dynawo/sources/Modeler/Common/DYNRingBuffer.h

```diff
@@ -53,6 +54,25 @@ class RingBuffer {
    */
   void add(double time, double value);
 
+  /**
+   * @brief Add timed value without performing other actions
+   *
+   * @param time timestamp of the value
+   * @param value value of the variable
+   */
+  void addNoCheck(double time, double value) {
+    queue_.push_back(std::make_pair(time, value));
```

Comment:
Would still be good to make sure that the time > latest time at least in debug

Reply:
Since it depends on the data in the file, either we perform no check (compliant with the name of the function) or we throw.

Reply:
yes throwing is OK

Reply:
maybe a better idea. Before adding the new delay when loading the parameters, check the timed values and return a parsing error, which leads to an exception.

Reply:
OK